src/libs/SidebarUtils.js

```diff
@@ -136,7 +131,7 @@ function getOrderedReportIDs() {
 
         const shouldFilterReportIfRead = hideReadReports && !ReportUtils.isUnread(report);
         const shouldFilterReport = shouldFilterReportIfEmpty || shouldFilterReportIfRead;
-        if (report.reportID.toString() !== currentlyViewedReportID
+        if (report.reportID.toString() !== Report.getCurrentlyViewedReportID()
```

Comment:
Since the SidebarLinks component now has access to the reportID directly from the route, I think I'd like to pass it as a parameter to `getOrderedReportIDs()`. 

That would only leave one reference to `Report.getCurrentlyViewedReportID()` (the one in User.js). I'm not sure if there is any other way for User.js to access that reportID :(

I'm just thinking if there is a way to get rid of it entirely.

Reply:
Let me think about that for a bit. I had a version where we were passing that more explicitly and then switched to it being better to do 2 things consistently (but less ideal) than 1 right thing and 1 wrong thing.

Reply:
My solution here ended up being to pass the params + parse the reportID from the navigation state for the one case where we needed that (and weren't triggering an action from a component). Both cases are using the reportID from the route as the "source of truth" so solves it for me, but lmk if you agree.